app.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+const app = express();
+const usersList = require("./userService");
+
+app.use(express.static('public'));
+
+app.get("/", function(req, res) {
+    res.render("index.pug");
+});
+
+// Load page
+app.get("/usersCenter", function(req, res) {
+    res.render("usersCenter.pug");
+});
+
+// Get Users
+app.get("/users", function(req, res) {
+    usersList.getUsers(req.query, res);
+});
+
+// Create a User
+app.post("/users", function(req, res) {
+    res.sendStatus(200);
+    req.on('data', function(c) {
+        var objectUser = JSON.parse(c.toString());
```

Comment:
use middlewares to parse body https://medium.com/@mmajdanski/express-body-parser-and-why-may-not-need-it-335803cd048c

Reply:
Could you please clarify what do you mean? Where should I use middleware?

Reply:
Fixed.